### DISCUSSION TITLE
【Step-2】请各位队长开始召集队友！

### DISCUSSION THREAD

**Discussion by GreyZeng** [FZUSESPR21/Class_Resources#4] [General]
1. 进入各自的Team中，点击如下位置：
![image](https://user-images.githubusercontent.com/12782103/112307574-16507f00-8cdc-11eb-8fc3-22caf310431f.png)

2. 把自己队伍中列表中的队友一一邀请进来：

![image](https://user-images.githubusercontent.com/12782103/112307693-37b16b00-8cdc-11eb-9176-02389d277776.png)

3. 点击+后，会出现如下弹框，在弹框中输入队友的用户名再点击+，即可邀请队友：

![image](https://user-images.githubusercontent.com/12782103/112308218-db9b1680-8cdc-11eb-862c-c8a827f78be3.png)

4. 把队友都邀请完毕后，请提醒队友赶紧接受邀请并加入队伍。

5. 最后要达到的效果是：
![image](https://user-images.githubusercontent.com/12782103/112400162-b6d98a00-8d42-11eb-9a79-b57f69ce26fb.png)

6. 请队长告知团队所有人（包括队长自己）在这个页面：https://github.com/orgs/FZUSESPR21/people

把自己的权限设置为public：

![image](https://user-images.githubusercontent.com/12782103/112310350-7268d280-8cdf-11eb-99eb-e054ce948639.png)

6. 以上都完成后，请队长在这个帖子下留下一条个性化的留言：比如：全军出发。。。准备就绪。。。。。any way，只要不是敏感词都可以。:)

以上任务，截止时间：2021年3月25日晚上10点

**逾期或者有队友迟迟没有加入进来，会扣团队分！** 

**注意：你们是一个团队，团队荣誉很重要！请队友也不要拖后腿。**

**Comment by Honghonglin** [comment]
**欢迎来到软工实践，全军出击**

**Comment by Lyr-GW** [comment]
烤盐人的软工实践！开冲💪

**Comment by hb594709591** [comment]
与其感慨路难行，不如马上出发。

**Comment by fzu-kwz** [comment]
老九门开冲！

**Reply by GreyZeng** [reply]
注意第6步

**Comment by LinxuanZheng** [comment]
"一条个性化的留言"

**Comment by gz321** [comment]
准备就绪

**Comment by Fzuerzmj** [comment]
逐梦校友圈完成任务

**Comment by Hao23333** [comment]
字节乱动冲冲冲

**Comment by Tozzger** [comment]
group->init()->done();
